lib/pronto/bundler_audit/gemfile_lock/scanner.rb

```diff
@@ -37,41 +37,14 @@ def call
         def find_relevant_line
           return unless (found_line_number = determine_line_number)
 
-          build_pronto_git_line(found_line_number)
+          found_line_number
```

Comment:
👍 But looks like the guard clause can go away now too. In fact we can probably just call `determine_line_number` instead of `find_relevant_line`.